lms/models/grading_info.py

```diff
@@ -69,3 +69,7 @@ class GradingInfo(BASE):
     tool_consumer_info_product_family_code = sa.Column(sa.UnicodeText(), nullable=True)
     h_username = sa.Column(sa.UnicodeText(), nullable=False)
     h_display_name = sa.Column(sa.UnicodeText(), nullable=False)
+
+    lms_id = sa.Column(
```

Comment:
Nullable, until (if) the existing data can be backfilled

Reply:
If we're deciding to group everything by `ApplicationInstance` and not by `LMS` I think we probably don't want to add this column here. `GradingInfo` already has an `oauth_consumer_key` column that is part of a unique constraint which groups it by `ApplicationInstance`